### ISSUE TITLE
Firefox does not respect aria-labelledby in elements named from content

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2373]
The link in the following code snippet has an empty accessible name, despite both the `img` and `svg` elements having an accessible name:

```html
<a href="#">
	<img src="https://via.placeholder.com/100" aria-labelledby="outside" />
	<svg role="img" aria-labelledby="circle" height="100">
		<title id="circle">Hello</title>
		<circle cx="50" cy="50" r="40" stroke="black" />
	</svg>
</a>
<div id="outside">Some text</div>
```

I tested this with buttons too, and it has the same problem. I suspect this will hold for all named from content elements (needs to be confirmed). This is such a substantial problem I think we should consider adding a check or rule to fail this.

Filed a ticket with Mozilla for it here: https://bugzilla.mozilla.org/show_bug.cgi?id=1652712
